config/services.yaml

```diff
@@ -234,6 +234,9 @@ services:
             $logoutUrl: '%app.idp.endpoint.logout%'
             $baseUri: '%app.base_url%'
 
+    App\Mealz\UserBundle\EventListener\InteractiveLoginListener:
```

Comment:
Shall we also use event subscriber rather than listener? Ref. https://tomasvotruba.com/blog/2019/05/16/don-t-ever-use-listeners/